woocommerce/woocommerce-ios
# Edit Order Address: Enable `Done` button when `Use as {Shipping/Billing} Address` toggle is turned on.

Closes #5148 

# Description

Currently, the `Use as {Shipping/Billing} Address` toggle in "Edit Order Address screen" does not enable the Done button when turned on.
This has been identified and reported in #5148.

This PR enables the `Done` button when `Use as {Shipping/Billing} Address` toggle is turned on. 

# Demo
https://user-images.githubusercontent.com/524475/138209674-8543baab-d796-40f6-aa5f-222b718d4d9d.mov

# Testing
1. Make sure that there is a processing order with Billing and Shipping addresses.
1. Go to Orders tab and open the order.
1. Tap the pencil icon next to the shipping address to edit it.
1. Turn on the `Use as Billing Address` toggle and observe that `Done` button is enabled.
1. Follow the same steps from 1 to 4, for editing billing address as well, and observe `Use as Shipping Address` toggle and `Done` button behave in the same manner as described above.

Update release notes:
